src/files/routes.rs

```diff
@@ -80,7 +153,15 @@ async fn stream_to_file(
 
 fn upload_file_docs(op: TransformOperation) -> TransformOperation {
     op.description("Upload a file.")
-        .response::<201, Json<FileUploaded>>()
-        // TODO: what is correct way to handle/reprort error docs in upload?
-        .response::<500, Json<FileUploaded>>()
+        .response::<201, Json<FileWrapper>>()
+        // TODO: what is correct way to handle/report error docs in upload?
+        .response::<500, Json<FileWrapper>>()
+}
+
+// TODO correct docs
+fn download_file_docs(op: TransformOperation) -> TransformOperation {
+    op.description("Download a file.")
+        .response::<206, String>()
```

Comment:
same comment as above: this feels incomplete. Perhaps it should also have a 401 (authorization)? Anything else?
I suppose this code (outside of the official server) may not be doing session management but I would suggest including it anyway.